main.py

```diff
@@ -92,14 +93,22 @@ def main_fun(num):
         path = input()
         open_file(path)
     else:
-        print("OK")
+        print("Something is wrong")
 
 
 if __name__ == '__main__':
     view_menu()
 
-    while 1:
+    while True:
```

Comment:
User should have a menu view before each choice, not only at the beginning of the application usage. For example: If I want to create 4 directories, then 2 files - I might forgott which number to choose to create a file and which to create a directory.